e2e/playwright/tests/release_hold.spec.ts

```diff
@@ -0,0 +1,26 @@
+import {test, expect} from '@e2e-support/test_fixture';
+import LegalHoldPluginPage from '../pages/legal_hold_plugin';
+
+test('Admin can remove legal hold successfully', async ({pw, pages}) => {
```

Comment:
I know you have the test cases in Confluence. Using that, can you submit a PR to https://github.com/mattermost/mattermost-test-management so all of the test cases are going to be added in our test management system (Zephyr Scale) and each will have a corresponding test ID (ex. `MM-Tnnn`). Then here, we can reference the test case ID in the title, like:
```js
test('MM-Tnnn Admin can remove legal hold successfully', ...)
```

Let me know if you bump into issues in creating/submitting a PR.

Reply:
ok

Reply:
I don't this is done yet but can be updated in a separate PR.

Reply:
No, i'm still working on this, i plan to complete it today.